File: src/GCov/Messaging/CoverageProcessEnded.java
package GCov.Messaging;

import com.intellij.util.messages.Topic;

public interface CoverageProcessEnded {
    Topic<CoverageProcessEnded> GCOVERAGE_RUN_ENDED_TOPIC = Topic.create("CoverageProcessEnded", CoverageProcessEnded.class);

    void ended(String buildDirectory);
}

File: src/main/kotlin/gcov/messaging/CoverageProcessEnded.kt
package gcov.messaging

import com.intellij.util.messages.Topic
import com.jetbrains.cidr.cpp.toolchains.CPPToolchains

interface CoverageProcessEnded {

    fun ended(buildDirectory: String, toolchain: CPPToolchains.Toolchain)

    companion object {
        val GCOVERAGE_RUN_ENDED_TOPIC = Topic.create("CoverageProcessEnded", CoverageProcessEnded::class.java)
    }
}
